matroid/src/detectors.py

```diff
@@ -216,3 +217,74 @@ def list_detectors(self):
   except Exception as e:
     raise error.APIConnectionError(message=e)
 
+
+@api_call(error.InvalidQueryError)
+def add_feedback_from_file(self, detectorId, filePath, feedback):
+  """Add feedback to a detector using a local file"""
+  (endpoint, method) = self.endpoints['add_feedback_from_file']
+  endpoint = endpoint.replace(':detector_id', detectorId)
+
+  data = {
+    'feedback': format_feedback(feedback)
+  }
+
+  try:
+    image_file = self.filereader.get_file(filePath)
+    files = {'file': image_file }
+    headers = {'Authorization': self.token.authorization_header()}
+
+    return requests.request(method, endpoint, **{'headers': headers, 'files': files, 'data': data})
+  except IOError as e:
+    raise e
+  except error.InvalidQueryError as e:
+    raise e
+  except Exception as e:
+    raise error.APIConnectionError(message=e)
+  finally:
+    if isinstance(image_file, io.IOBase):
+      image_file.close()
+
+
+@api_call(error.InvalidQueryError)
+def add_feedback_from_url(self, detectorId, imageURL, feedback):
+  """Add feedback to a detector using an image URL"""
+  (endpoint, method) = self.endpoints['add_feedback_from_url']
+  endpoint = endpoint.replace(':detector_id', detectorId)
+
+  data = {
+    'feedback': format_feedback(feedback),
+    'url': imageURL
+  }
+
+  try:
+    headers = {'Authorization': self.token.authorization_header()}
+    return requests.request(method, endpoint, **{'headers': headers, 'data': data})
+  except error.InvalidQueryError as e:
+    raise e
+  except Exception as e:
+    raise error.APIConnectionError(message=e)
```

Comment:
Same comment as on the Node PR about whether `add_feedback_from_file` and `add_feedback_from_url` should be one method